templates/content/pattern-field-list--field-group--group-organisation--node--oe-person--teaser.html.twig

```diff
@@ -0,0 +1,10 @@
+{#
+/**
+ * @file
+ * Extends field list pattern.
+ */
+#}
+{% extends '@oe_theme/patterns/field_list/pattern-field-list.html.twig' %}
+{% block content %}
+  <div class="ecl-u-mb-m">{{ parent() }}</div>
```

Comment:
Why do we need to add this? The list item where this is rendered already has a ecl-u-mb-s margin.

Reply:
<img width="802" alt="Screenshot 2021-03-10 at 20 31 52" src="https://user-images.githubusercontent.com/1183951/110671827-04e97c00-81e0-11eb-9d3f-f04dfc1a38df.png">

We need it to keep height between lines the same. Structure of elements is here /admin/structure/types/manage/oe_person/display/teaser